lib/terminus.js

```diff
@@ -125,6 +129,9 @@ function terminus (server, options = {}) {
     })
   }
 
+  // add the healthcheck to the internal states
+  internal.healthChecks = healthChecks
```

Comment:
If calling createTerminus multiple times, wouldn't this just override the last health checks?

Reply:
oh, you are right 🤔

Reply:
reopening, sorry for closing this convo

Reply:
No prob haha :)
I really do not think we get around a class as I proposed in my PR..